docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -77,7 +78,7 @@ Configuration API::
 public class AxonConfig {
     // omitting other configuration methods...
     public void configureProcessorDefault(EventProcessingConfigurer processingConfigurer) {
-        processingConfigurer.usingTrackingEventProcessors();
+        processingConfigurer.usingPooledStreamingEventProcessors();
```

Comment:
This configuration block is incorrect. We don't have the `usingPooledStreamingEventProcessors` method anymore at all.